File: Assignment7/Program3.c

//Write a program to find  factorial of given Number 

//Input : 5
//Output : 120 (5*4*3*2*1 ) //5(5-1)*(5-2)*(5-3)(5-4)

//Input : -5
//Output : 120

//Input : 4
//Output : 24

int Factorial(int iNo)
{ 
     auto int icnt = 0;
     auto int fact = 1;

     if(iNo < 0)
     {
         iNo = -iNo;
     }
     for(icnt = 1 ; icnt <= iNo; icnt++)
     {
        fact = fact * icnt;
          
                 
     }

    return fact;
     
    }


#include<stdio.h>



int main()
{
   auto int iValue = 0;
   auto int iRet = 0;

   printf("Enter Number:");
   scanf("%d",&iValue);

   
    iRet = Factorial(iValue);

 printf("Factorial of Number is :%d ",iRet);

    return 0;
}

File: Assignment35/Program3.java
 
 //Write a program which accept N numbers from user and accept one another number as NO ,return  index of first Occurance of that number 

  //Input : 6
 //No : 66

 //Elements : 85 66 3 66 93 88
 //Output : 1

 //Input : 6
 //No : 12

 //Elements : 85 11 3 15 11 111
 //Output : -1

 import java.util.*;




 class Program3
 {  
    public static int FirstOcc(int Arr[] , int iNo)
    {
         int iCnt1 = 1;
         int iCnt =0;
         int ipos = -1;
       for(iCnt = 0; iCnt < Arr.length; iCnt++)
       {
          if(Arr[iCnt] == iNo)
          {  
              
           ipos = iCnt1;
           
          }
          iCnt1++;   
          
       }
      return ipos;
    } 

    public static void main(String Args[])
    {
      Scanner sobj = new Scanner(System.in);
      int iSize =0 ,iValue = 0;
      int iCnt = 0 ,iRet =0 ;

      System.out.println("Enter number of elements :\n");
      iSize = sobj.nextInt();
      
      int Brr[] = new int[iSize];

      System.out.println("Enter Number :");
      iValue = sobj.nextInt();
    
      System.out.println("Enter Elements :");
      for(iCnt = 0; iCnt < iSize; iCnt++)
      {
          Brr[iCnt] = sobj.nextInt(); 
      } 
        
      iRet = FirstOcc(Brr , iValue);
      System.out.println(" : "+iRet);

    }
 }